client/my-sites/plans/current-plan/main.jsx

```diff
@@ -140,7 +142,11 @@ const PlanDetailsComponent = React.createClass( {
 						isPlaceholder={ false } />
 				</Card>
 				{ selectedSite &&
-					<Card href={ '/plans/compare/' + selectedSite.slug }>
+					<Card
+						href={
+							( showPlanFeatures ? '/plans/' : '/plans/compare/' ) + selectedSite.slug
```

Comment:
Minor, but I'd prefer string interpolation here to avoid potential bugs with concatenation. 
```
showPlanFeatures ? `/plans/${ selectedSite.slug }` : `/plans/compare/${ selectedSite.slug }`
```